channels/vn.m3u

```diff
@@ -1,130 +1,1075 @@
+#EXTINF:-1 tvg-id="DongNai2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Nai 2 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sda/dongnai2_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="DongNai2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Nai 2 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/dongnai2_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="DongNai2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Nai 2 (576p)
+http://118.107.85.4:1935/live/smil:DNTV2.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="DongThap1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Tháp 1 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sdc/dongthap_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="DongThap1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Tháp 1 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sdc/dongthap_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="DongThap1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Tháp 1 (540p)
+http://202.43.109.142:1935/THDT/thdttv/playlist.m3u8
+#EXTINF:-1 tvg-id="DongThap1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Tháp 1 (720p) [Geo-blocked]
+https://htv-drm-live-cdn.fptplay.net/CDN-FPT02/DONGTHAP-SD-ABR/playlist.m3u8
+#EXTINF:-1 tvg-id="DongThap2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Đồng Tháp 2 (540p)
+http://202.43.109.145:1935/thdt2/thdt2/playlist.m3u8
+#EXTINF:-1 tvg-id="GiaLai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Gia Lai (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sda/gialai_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GiaLai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Gia Lai (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/gialai_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GiaLai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Gia Lai (486p)
 http://113.161.25.3:8134/hls/gialaitv/gialaitv.m3u8
-#EXTINF:-1 tvg-id="GoodIdeaTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Good Idea TV (720p)
-https://livefta.malimarcdn.com/ftaedge00/gooditv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="HBTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://i.imgur.com/BnFWgS7.png" group-title="",HBTV (406p) [Not 24/7]
-http://hoabinhtvlive.746b3ddb.cdnviet.com/hoabinhtv/playlist.m3u8
```

Comment:
Working link

Reply:
#EXTINF:-1 tvg-id="GoodIdeaTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Good Idea TV (720p)
https://livefta.malimarcdn.com/ftaedge00/gooditv.sdp/playlist.m3u8
In #5293 

#EXTINF:-1 tvg-id="HBTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://i.imgur.com/BnFWgS7.png" group-title="",HBTV (406p) [Not 24/7]
http://hoabinhtvlive.746b3ddb.cdnviet.com/hoabinhtv/playlist.m3u8
Exitsted

Reply:
> Exitsted

?

Reply:
> > Exitsted
> 
> ?

#EXTINF:-1 tvg-id="HoaBinh.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Hòa Bình (406p)
http://hoabinhtvlive.746b3ddb.cdnviet.com/hoabinhtv/chunklist.m3u8